src/Magnum/Math/Intersection.h

```diff
@@ -170,6 +170,22 @@ for plane normal @f$ \boldsymbol n @f$ and determinant @f$ w @f$.
 */
 template<class T> bool rangeFrustum(const Range3D<T>& range, const Frustum<T>& frustum);
 
+/**
+@brief Intersection of a ray with a range.
+@param rayOrigin    Origin of the ray
+@param invRayDir  Componentwise inverse of the ray direction
+@param range Range
+@return @cpp true @ce if the the ray intersect the range, @cpp false @ce otherwise.
+
+Note that you need to pass the inverse ray direction and not the ray direction.
+The purpose for this is to reduce the number of times you have to compute
+a ray inverse, when doing multiple ray range intersections (For example
+when traversing an aabb-tree).
+The algorithm implemented is a version of the classical slabs algorithm, introduced by
+[Kay and Kajiya](http://papers.cumincad.org/data/works/att/67d2.content.pdf).
```

Comment:
Just for clarity, can you also reference the paper from JCGT here? Since it's basically a 1:1 port of it.